src/UKHO.ERPFacade.Common/Logging/EventIds.cs

```diff
@@ -477,7 +477,17 @@ public enum EventIds
         /// <summary>
         /// 940095 - Shared API Key Configuration is missing.
         /// </summary>
-        SharedApiKeyConfigurationMissing = 940095
+        SharedApiKeyConfigurationMissing = 940095,
+
+        /// <summary>
+        /// 940096 - S100 Request to SAP failed.
+        /// </summary>
+        S100RequestToSapFailedException = 940096,
+
+        /// <summary>
+        /// 940097 - S100 data content sent to SAP.
```

Comment:
```suggestion
        /// 940097 - S-100 data content sent to SAP.
```

Reply:
Please update the "S-100" across the project as per given naming convention wherever applicable.

Reply:
Updated the "S-100" in EventIds.cs